`make check-licenses` fails

```
% make check-licenses
go install github.com/elastic/go-licenser@v0.4.0
go run github.com/elastic/go-licenser@v0.4.0 -d .
package github.com/elastic/go-licenser@v0.4.0: can only use path@version syntax with 'go get' and 'go install' in module-aware mode
make: *** [check-licenses] Error 1
```

My golang ignorance means I don't immediately know what to do about "module-aware mode".

I'm using go 1.16:

```
% go version
go version go1.16.3 darwin/amd64
```

which I think is acceptable given `go 1.14` in "go.mod".  What is the min golang version for the extension?

--- a/.ci/Jenkinsfile
+++ b/.ci/Jenkinsfile
@@ -63,7 +63,7 @@ pipeline {
           }
           axis {
             name 'GO_VERSION'
-            values '1.15', '1.16', '1.17'
+            values '1.17', 1.18
           }
         }
         stages {

--- a/apm-lambda-extension/go.mod
+++ b/apm-lambda-extension/go.mod
@@ -3,14 +3,21 @@ module elastic/apm-lambda-extension
 go 1.17
 
 require (
-	github.com/google/go-cmp v0.5.6 // indirect
 	github.com/google/uuid v1.3.0
 	github.com/joho/godotenv v1.4.0
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0
 	go.elastic.co/ecslogrus v1.0.0
+	gotest.tools v2.2.0+incompatible
+)
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/google/go-cmp v0.5.6 // indirect
+	github.com/magefile/mage v1.9.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
 	golang.org/x/sys v0.0.0-20211102192858-4dd72447c267 // indirect
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
-	gotest.tools v2.2.0+incompatible
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
 )

--- a/apm-lambda-extension/go.mod
+++ b/apm-lambda-extension/go.mod
@@ -1,6 +1,6 @@
 module elastic/apm-lambda-extension
 
-go 1.14
+go 1.17
 
 require (
 	github.com/google/go-cmp v0.5.6 // indirect

